### ISSUE TITLE
Make consortium name configurable

### ISSUE THREAD

**Issue by nikolas-digitalBabylon** [hyperledger-labs/minifabric#286]
Currently consortium name is "SampleConsortium" - I do not see a way to configure consortium name. This should be configurable by the spec file.

**Comment by litong01** [comment]
Yes. It is not at present. You are very welcome to submit a PR to fix this should be straight forward.

**Comment by davidkhala** [comment]
@litong01 in design thinking, should we suggest the non-system channel fashion as recommended way? If so, we can skip it feature request and move forward.

**Comment by nikolas-digitalBabylon** [comment]
> @litong01 in design thinking, should we suggest the non-system channel fashion as recommended way? If so, we can skip it feature request and move forward.

I would say this makes sense, since the system channel actually is only a "baggage" and is not needed anymore.
